src/main/java/de/internetsicherheit/brl/bloxberg/cache/ethereum/ReducedTransObjectArray.java

```diff
@@ -0,0 +1,29 @@
+package de.internetsicherheit.brl.bloxberg.cache.ethereum;
+
+import org.web3j.protocol.core.methods.response.Transaction;
+
+import java.util.ArrayList;
+
+public class ReducedTransObjectArray {
```

Comment:
This file and `ReducedTransObject` class were part of a former PR. Is this needed in this PR?